### PR TITLE
[airbyte-ci] Improved PR generation flow and a bugfix

### PR DESCRIPTION
**Pull request by natikgadzhi**
## What

A batch of changes to `airbyte-ci` that @bleonard and I did to push up the poetry + base image migration PRs.

These seem to make sense universally, so I'm pushing them up:
- Don't build connector image on poetry migration (it will be built in CI anyway)
- Don't create a PR in `up_to_date` until asked to.
- Bugfix in `pull_request --bump`

## TODO
This still needs a version bump to airbyte-ci itself.

## Can this PR be safely reverted and rolled back?
- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

File: airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/up_to_date/commands.py [hunk 1]
```diff
@@ -42,18 +42,23 @@
     default=False,
     help="Create a pull request.",
 )
-
+@click.option(
+    "--no-bump",
```
Comment (girarda) [inline airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/up_to_date/commands.py:46]: nit: the negative flag is a little confusing. why not `--bump`?

**Review by girarda — APPROVED** [review]
looks good thanks for fixing!

File: airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/up_to_date/commands.py (hunk 1 above)
Comment (natikgadzhi) [reply airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/up_to_date/commands.py:46]: I had the same thought, but honestly, good enough for an internal tool for now.

**Comment by natikgadzhi** [comment]
One type check failed but given there are 593 of them total and I can't run just one file locally, I'm sorry, I'm merging this so I can easily document how to regen all those PRs.

**Pull request merged**